deploy-service/teletraanservice/src/test/java/com/pinterest/teletraan/resource/ResourceAuthorizationTest.java

```diff
@@ -64,6 +64,7 @@
 @ExtendWith(DropwizardExtensionsSupport.class)
 public class ResourceAuthorizationTest {
     private static final String SCRIPT_TOKEN = "script_token";
+    private static final String PINGER_TOKEN = "pinger_token";
```

Comment:
what is pinger token?

Reply:
This is just a string to represent a pinger token.

Reply:
```java
            when(tokenRolesDAO.getByToken(PINGER_TOKEN)).thenReturn(pingerRolesBean);
```